app/views/experience_stages/show.html.erb

```diff
@@ -43,7 +43,8 @@
     <div class='row'>
       <% @themes.each do |theme| %>
         <div class='col-sm-3 nav-link' href='#theme-<%= theme.id %>' du-smooth-scroll du-scrollspy offset='110'>
-          <p class='title'><%= theme.title.html_safe %></p>
+          <p class='title lg'><%= theme.title.html_safe %></p>
```

Comment:
Instead of defining a ton of extra css, you can just add the classes in line as: `hidden-sm`